dev-libs/udis86/udis86-1.7.2.recipe

```diff
@@ -6,11 +6,11 @@ HOMEPAGE="http://udis86.sourceforge.net/"
 COPYRIGHT="2002-2012, Vivek Thampi"
 LICENSE="BSD (2-clause)"
 REVISION="1"
-SOURCE_URI="https://github.com/vmt/udis86/archive/v1.7.2.tar.gz"
+SOURCE_URI="https://github.com/vmt/udis86/archive/v$portVersion.tar.gz"
 CHECKSUM_SHA256="43567f7e12168943c5b5ffb3d3f5b7a33cb36328f8938a993458f3ded0ba5779"
```

Comment:
```diff
 SOURCE_URI="https://github.com/vmt/udis86/archive/v$portVersion.tar.gz" 
 CHECKSUM_SHA256="43567f7e12168943c5b5ffb3d3f5b7a33cb36328f8938a993458f3ded0ba5779"
+SOURCE_FILENAME="udis86-$portVersion.tar.gz"
 PATCHES="udis86-$portVersion.patchset"
```
(It would be nice to have a `SOURCE_FILENAME`.)